### DISCUSSION TITLE
How to extract power metering from plug TS0121

### DISCUSSION THREAD

**Discussion by syfq91** [arendst/Tasmota#13171] [Q&A] [answered]
Hi,

I have a power metering plug [TS0121](https://zigbee.blakadder.com/Tuya_TS0121.html) and would like to know how to get the power metering reading.

I was able to get the power metering reading while using zigbee2mqtt but it is not showing the reading while using tasmota2zigbee. Is there any setting that I have to enable? I am using Sonoff ZbBridge.

Below is the the result of zbstatus3:
`20:52:05.149 MQT: stat/zbBridge/RESULT = {"ZbStatus3":[{"Device":"0x6D0D","Name":" powerPlug","IEEEAddr":"0x60A423FFFE7FF62B","ModelId":"TS0121","Manufacturer":"_TZ3000_8nkb7mof","Endpoints":[1],"Config":["O01"],"Power":1,"Reachable":true,"LastSeen":1916,"LastSeenEpoch":1632054009,"LinkQuality":29}]}`

Thanks for the help.

**Comment by ascillato** [comment, marked as answer]
Hi, as you didn't have any answer here, you can try addressing this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4).
